beacon_node/beacon_chain/src/custody_context.rs

```diff
@@ -247,39 +253,92 @@ impl<E: EthSpec> CustodyContext<E> {
 
     /// Restore the custody context from disk.
     ///
-    /// * If NodeCustodyType::custody_count < validator_custody_at_head, it means the attached
-    ///   validate stake has increased the node's CGC. We ignore the CLI input.
-    /// * If NodeCustodyType::custody_count > validator_custody_at_head, it means the user has
-    ///   changed the node's custody type via either the --supernode or --semi-supernode flags,
-    ///   and will require a resync until we implement column backfill for this scenario.
+    /// # Behavior
+    /// * If [`NodeCustodyType::get_custody_count_override`] < validator_custody_at_head, it means the attached
+    ///   validator stake has increased the node's CGC. We ignore the CLI input.
```

Comment:
This comment is a little unclear. Maybe something along the lines of
```
    ///   the node has a higher contribution of custody from the number of validators attached compared to the one derived from cli flags
```
Something a little less verbose than that though :P

Reply:
Updated to
```rust
    /// * If [`NodeCustodyType::get_custody_count_override`] < validator_custody_at_head, it means
    ///   validators have increased the CGC beyond the derived CGC from cli flags. We ignore the CLI input.
```